docs/en/guides/developer/transactional.md

```diff
@@ -0,0 +1,50 @@
+# Transactional (ACID) support
+
+INSERTs into one partition in one table of the MergeTree\* family up to `max_insert_block_size` rows are transactional (ACID):
```

Comment:
Are you sure this formatting is enough to understand that next points are valid only if `up to `max_insert_block_size` rows` is true? 
Actually for ACI. D is valid in any case.

Reply:
Thanks @qoega , I updated, here is how it reads now.  Also moved the ACID lines from H2 to a dictionary:
![image](https://user-images.githubusercontent.com/25182304/178489460-73f0d18c-9ab2-40be-b2e5-99c6b6fc0dcc.png)